internal/testing/runchecks.sh

```diff
@@ -74,47 +74,47 @@ if [[ "${TRAVIS_OS_NAME:-}" == "linux" ]]; then
 fi
 
 result=0
-
 rootdir="$(pwd)"
+
+# Run Go tests.
```

Comment:
Move this comment to the loop or remove entirely. Add a separate comment to the test-summary line to explain what it's doing

Reply:
Done.

Reply:
Also removed the `|| result=1` for this line; if test-summary doesn't build, we should just exit.